python-sdk/tests/benchmark/settings.py

```diff
@@ -6,7 +6,7 @@
 )
 publish_benchmarks_schema = os.getenv("ASTRO_PUBLISH_BENCHMARK_SCHEMA", "benchmark")
 publish_benchmarks_table = os.getenv(
-    "ASTRO_PUBLISH_BENCHMARK_TABLE", "load_files_to_databaseV2"
+    "ASTRO_PUBLISH_BENCHMARK_TABLE", "load_files_to_databaseV3"
```

Comment:
We could also add this as an argument while running the job. WDYT?

Reply:
I think we won't be changing this much often once this is stable.

Reply:
But I'll add a backlog ticket for this

Reply:
here - https://github.com/astronomer/astro-sdk/issues/1009